tests/conftest.py

```diff
@@ -29,42 +29,27 @@
 
 pytest_plugins = ["pytester", "pytest_mock"]
 
-if sys.version_info < (3, 13):
-    if sys.platform == "win32":
-        EventLoop = asyncio.ProactorEventLoop
-    else:
-        EventLoop = asyncio.SelectorEventLoop
-else:
-    EventLoop = asyncio.EventLoop
-
-if sys.version_info >= (3, 12):
-
-    def eager_task_loop_factory() -> EventLoop:
-        loop = EventLoop()
-        loop.set_task_factory(asyncio.eager_task_factory)
-        return loop
-
-    eager_marks: list[pytest.MarkDecorator] = []
-else:
-    eager_task_loop_factory = EventLoop
-    eager_marks = [pytest.mark.skip(reason="eager tasks not supported yet")]
-
 asyncio_params = [
-    pytest.param(
-        ("asyncio", {"debug": True, "loop_factory": None}),
-        id="asyncio",
-    ),
+    pytest.param(("asyncio", {"debug": True}), id="asyncio"),
     pytest.param(
         ("asyncio", {"debug": True, "loop_factory": uvloop.new_event_loop}),
         marks=uvloop_marks,
         id="asyncio+uvloop",
     ),
-    pytest.param(
-        ("asyncio", {"debug": True, "loop_factory": eager_task_loop_factory}),
-        marks=eager_marks,
-        id="asyncio+eager",
-    ),
 ]
+if sys.version_info >= (3, 12):
+
+    def eager_task_loop_factory() -> asyncio.AbstractEventLoop:
+        loop = asyncio.new_event_loop()
```

Comment:
new_event_loop uses the deprecated policy it's best to use EventLoop on 3.13+

Reply:
the new_event_loop will be a legacy alias for EventLoop that's kept forever so I suppose it doesn't matter

Reply:
Where is this information coming from? I'm not seeing anything in the 3.13 or 3.14 docs about `new_event_loop()` using a deprecated policy. Link?

Reply:
https://github.com/python/cpython/blob/main/Lib/asyncio/events.py#L821 it calls get_event_loop_policy, which is deprecated. Eventually new_event_loop will be a deprecated alias for EventLoop